didcomm_resolver/role/requester/routes.py

```diff
@@ -191,7 +191,7 @@ async def connection_register(request: web.BaseRequest):
     session = await context.session()
     connection_id = request.match_info.get("conn_id")
     body = await request.json() if request.body_exists else {}
-    methods: Sequence[str] = body.get("methods", [])
+    methods: Set[str] = body.get("methods", set())
```

Comment:
These values will come in as a list from JSON since JSON doesn't have a concept of a set built in. To ensure there are actually no duplicates, we'll have to do:
```suggestion
    methods: Set[str] = set(body.get("methods", []))
```